src/test/java/com/ericsson/ec/subscription_object_test/SubscriptionObjectTest.java

```diff
@@ -0,0 +1,74 @@
+package com.ericsson.ec.subscription_object_test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.ericsson.ec.helpers.MediaType;
+import com.ericsson.ec.subscriptionobject.RestPostSubscriptionObject;
```

Comment:
Samer comment as in last file, I would recommend to use json.org.

Reply:
fixed